app/presenters/templates/components/form.latte

```diff
@@ -4,9 +4,17 @@
 </ul>
 
 <table>
-<tr n:foreach="$form->controls as $input" n:class="$input->required ? required">
-	<th>{label $input /}</th>
-	<td>{input $input} <span class=error n:ifcontent>{$input->error}</span></td>
-</tr>
+{foreach $form->controls as $input}
+    <tr n:if="!$input instanceof Nette\Forms\Controls\HiddenField" n:class="$input->required ? required">
```

Comment:
I've implemented a better solution to identify hidden fields https://github.com/nette/forms/commit/ffb3b6bd178f5157ff615ef2d73711427059a5e6, so something like `n:if="$input->getOption(type) !== hidden"` can be used.